packages/plugin-core/package.json

```diff
@@ -162,6 +162,13 @@
             }
         },
         "commands": [
+            {
+                "command": "dendron.pasteLink",
+                "title": "Dendron: paste link",
+                "desc": "Copy wiki link to note",
+                "docLink": "dendron.topic.commands.md",
+                "docPreview": "![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/command.copy-link.gif)"
```

Comment:
the docPreview is a copy/paste of the `copy-link` command. if we don't have a preview, then this should be blank